modules/dxp/apps/analytics/analytics-reports-rest-impl/src/main/java/com/liferay/analytics/reports/rest/internal/resource/v1_0/AssetMetricResourceImpl.java

```diff
@@ -32,22 +41,63 @@ public AssetMetric getGroupAssetMetric(
 			Integer rangeKey, String[] selectedMetrics)
 		throws Exception {
 
-		AnalyticsCloudClient analyticsCloudClient = new AnalyticsCloudClient(
-			_http);
+		List<Long> analyticsCloudChannelIds = new ArrayList<>();
 
 		Group group = _groupLocalService.getGroup(groupId);
 
+		DepotEntry depotEntry = _depotEntryLocalService.fetchGroupDepotEntry(
+			groupId);
+
+		if (depotEntry != null) {
+			analyticsCloudChannelIds = _getAnalyticsCloudChannelIds(
+				_depotEntryGroupRelLocalService.getDepotEntryGroupRels(
+					depotEntry));
+		}
+		else {
+			analyticsCloudChannelIds = Collections.singletonList(
+				Long.valueOf(
+					group.getTypeSettingsProperty("analyticsChannelId")));
+		}
+
+		AnalyticsCloudClient analyticsCloudClient = new AnalyticsCloudClient(
+			_http);
+
 		return analyticsCloudClient.getAssetMetric(
 			_analyticsSettingsManager.getAnalyticsConfiguration(
 				group.getCompanyId()),
-			assetId, assetType,
-			Long.valueOf(group.getTypeSettingsProperty("analyticsChannelId")),
-			identityType, rangeKey, selectedMetrics);
+			assetId, assetType, analyticsCloudChannelIds, identityType,
+			rangeKey, selectedMetrics);
+	}
+
+	private List<Long> _getAnalyticsCloudChannelIds(
+			List<DepotEntryGroupRel> depotEntryGroupRels)
+		throws Exception {
+
+		List<Long> analyticsCloudChannelIds = new ArrayList<>();
+
+		for (DepotEntryGroupRel depotEntryGroupRel : depotEntryGroupRels) {
+			Group group = _groupService.getGroup(
```

Comment:
Hey @marcosapmf, I tried to deploy it locally and got an error. Shouldn't it be:
```Group group = _groupLocalService.getGroup(```

Reply:
Oops... thanks for catching this, @interaminense :smile: